apps/contract-sync-job/README.md

```diff
@@ -0,0 +1,19 @@
+# Contract Sync Job
+
+## Summary
+The main purpose of this script is to sync the database with the Contract's events. The Contract's events and mappings are the main source of truth; hence, all events overwrite the existing records in database
+
+
+### Related Jobs: 
+- [Linear Sync Job](../linear-sync-job/README.md).
+- [Twitter Contribution Job](../twitter-contribution-job/README.md).
```

Comment:
I don't think we need this section. If anything maybe we group our jobs in a subdirectory